### ISSUE TITLE
Placeholder for data registry capability

### ISSUE THREAD

**Issue by alexanderdean** [snowplow/iglu#218]
This is based on @brapse's idea of SoundCloud's internal "Data AMT" service which provides a registry of what data lives where. This is distinct (but complementary to) a schema registry which governs your data types.

To give a Snowplow example, in the data registry you could record:

* Where the raw collector payloads live
* Where the enriched events live
* Where the shredded data lives in Redshift

It can be more than just locations of course - relevant metadata, provenance, etc

**Comment by andybritz** [comment]
Is this analogous to the LinkedIn [WhereHows](https://github.com/linkedin/WhereHows) project?

**Comment by alexanderdean** [comment]
Yes @andybritz ! It's funny because at [Crunch conf](http://www.crunchconf.com/) there was a LinkedIn talk straight after @brapse's which talked about WhereHows.

We may be able to just use WhereHows instead of build this into Iglu - it needs further investigation...

**Comment by gregbonnette** [comment]
May be worth looking at Atlas too.

http://atlas.incubator.apache.org/

>

-- 

<http://www.ironsidegroup.com/?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>


Greg Bonnette
Vice President, Strategy & Innovation

O  781-860-8840 x256
M  857-488-4478
gbonnette@ironsidegroup.com
http://ironsidegroup.com
<http://ironsidegroup.com?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
Minding your business™
<https://www.facebook.com/ironsidegroup?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<https://twitter.com/ironsidegroup?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<https://plus.google.com/+Ironsidegroup/posts?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<http://www.linkedin.com/company/the-ironside-group-inc.?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>
<https://www.youtube.com/user/IronsideGroup?utm_source=WiseStamp&utm_medium=email&utm_term=&utm_content=&utm_campaign=signature>

**Comment by alexanderdean** [comment]
Yes, have been meaning to take a look at Atlas! Thanks Greg.
